instructions/housing/3-MachineLearning.md

```diff
@@ -47,21 +47,24 @@ able to find those parameters and build that particular model.
 
 4) Go back Jenkins and observe that the model succeeded. Note that 
 it was identified as an experiment due to using non-default parameters. 
-And so, the pipeline skipped the production check.
+The pipeline therefore skipped the production 'register model' step and ran the experiment step instead.
 
 ![GreenExperiment](../images/GreenExperiment.png)
 
+This lets us use Jenkins like an [ML training orchestration platform](https://github.com/EthicalML/awesome-production-machine-learning#model-training-orchestration).
+
```

Comment:
This note helps to clarify that our use of Jenkins here is just an example/illustration